api/elections.go

```diff
@@ -661,8 +662,12 @@ func (a *API) electionFilterPaginatedHandler(msg *apirest.APIdata, ctx *httprout
 		}
 	}
 	page = page * MaxPageSize
+
 	// get election list
-	elections, err := a.indexer.ProcessList(body.OrganizationID, page, MaxPageSize, body.ElectionID.String(), 0, 0, body.Status, body.WithResults)
+	if body.WithResults == nil {
+		body.WithResults = new(bool)
+	}
+	elections, err := a.indexer.ProcessList(body.OrganizationID, page, MaxPageSize, body.ElectionID.String(), 0, 0, body.Status, *body.WithResults)
```

Comment:
This is a 144 chars line, it should be broken. Please try not to surpass the 120.